app/pages/event/[event_id_or_slug]/index.js

```diff
@@ -0,0 +1,3 @@
+import Event from 'Containers/Event';
```

Comment:
should be `/events/[id_or_slug].js`. Update the URLs accordingly elsewhere.

For using the useModals which internally uses the ModalManager context, you need to wrap the component with createPage which will do it for you. You'll need to pass a query to create page which fetches the viewer.

Reply:
got it @hereisnaman. I will complete it today

Reply:
> should be `/events/[id_or_slug].js`. Update the URLs accordingly elsewhere.
> 
> For using the useModals which internally uses the ModalManager context, you need to wrap the component with createPage which will do it for you. You'll need to pass a query to create page which fetches the viewer.

but createPage is already using useViewer hook.
 also in createPage when i pass event page i don't need to pass any query because event query id done by viewEvent modal i just need to file id/slug based on URL right?

Reply:
Learn for useViewer functions. It doesn't make any query to the API.